src/helpers/index.js

```diff
@@ -66,17 +95,21 @@ const logComponentType = (selected) => (
     )).join('  ')
 );
 
-module.exports.logIntro = ({ name, dir, type }) => {
+module.exports.logIntro = ({ name, dir, type, styling }) => {
   console.info('\n');
   console.info(`✨  Creating the ${chalk.bold.rgb(...colors.gold)(name)} component ✨`);
   console.info('\n');
 
 
   const pathString = chalk.bold.rgb(...colors.blue)(dir);
-  const typeString = logComponentType(type);
+  const componentTypeString = logComponentType(type);
+  const stylingTypeString = logStylingType(styling);
 
   console.info(`Directory:  ${pathString}`);
-  console.info(`Type:       ${typeString}`);
+  console.info(`Type:       ${componentTypeString}`);
+  if (styling && styling !== 'none') {
```

Comment:
I talked more about this above, but yeah I think we should log the styling even when "none" is selected :)